src/components/CompareButton.js

```diff
@@ -1,11 +0,0 @@
-import React from 'react';
```

Comment:
This component was removed as a result of adding the `Button` component.  This used to trigger the API call to compare currencies.  It is all possible through the new component instead.